cluster/src/main/java/org/apache/iotdb/cluster/log/applier/BaseApplier.java

```diff
@@ -219,10 +219,9 @@ private void processPlanWithTolerance(InsertPlan plan, DataGroupMember dataGroup
           pullTimeseriesSchema(plan, dataGroupMember.getHeader());
           plan.recoverFromFailure();
           getQueryExecutor().processNonQuery(plan);
-        } else {
-          throw e;
         }
-      } else throw e;
+      }
+      throw e;
```

Comment:
Maybe these two `throws` is difficult to merge, but there's definitely something wrong with this `throws` ? so will it be thrown again when it executes correctly?